perun-core/src/test/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_group_attribute_def_def_adGroupNameTest.java

```diff
@@ -37,4 +37,12 @@ public void testCorrectSyntax() throws Exception {
 
 		classInstance.checkAttributeSyntax((PerunSessionImpl) sess, group, attributeToCheck);
 	}
+
+	@Test
+	public void testCorrectSyntaxWithDash() throws Exception {
+		System.out.println("testCorrectSyntax()");
+		attributeToCheck.setValue("correct-Value-with-Dash");
+
+		classInstance.checkAttributeSyntax((PerunSessionImpl) sess, group, attributeToCheck);
```

Comment:
For a better readability, you can use a static method `assertThatNoException().isThrownBy()` from the assertJ library. That way, it is more clear what is the purpose of this test.